apcd-cms/src/client/src/components/Admin/EditExceptionModal/EditExceptionModal.tsx

```diff
@@ -233,20 +229,27 @@ const EditExceptionModal: React.FC<EditRecordModalProps> = ({
         className="modal-dialog modal-lg"
         onClosed={dismissMessages}
       >
-        <ModalHeader close={closeBtn}>
-          Edit Exception ID {exception.exception_id} for {exception.entity_name}
-        </ModalHeader>
+        <div className="modal-header">
+          <h4 className="modal-title text-capitalize">
+            Edit Exception ID {exception.exception_id} for{' '}
+            {exception.entity_name}
+          </h4>
+          <button type="button" className="close" onClick={onClose}>
+            <span aria-hidden="true">&#xe912;</span>
+          </button>
+        </div>
```

Comment:
Can we go back to this, I think it was a merge issue. Without this the close button will not work.
```suggestion
        <ModalHeader close={closeBtn}>
          Edit Exception ID {exception.exception_id} for {exception.entity_name}
        </ModalHeader>
```